mtags/src/main/scala-2/scala/meta/internal/pc/MetalsGlobal.scala

```diff
@@ -123,7 +123,11 @@ class MetalsGlobal(
 
   def semanticdbSymbol(symbol: Symbol): String = {
     import semanticdbOps._
-    val semantic = symbol.toSemantic
+    val semantic =
+      if (symbol.isLocallyDefined)
+        // local symbols could have the same id assigned
+        this.synchronized(symbol.toSemantic)
```

Comment:
I wonder how bad it would be to have this here. The alternative is to copy symbol ops and make them thread safe